src/MakePotCommand.php

```diff
@@ -628,7 +628,7 @@ protected function extract_strings() {
 
 			if ( $this->main_file_path && $this->location ) {
 				$translation->addReference(
-					str_replace( "$this->source/", '', $this->main_file_path )
+					str_replace( "$this->source/", '', Utils\normalize_path( $this->main_file_path ) )
```

Comment:
In other places we use `ltrim` as well:

```suggestion
					ltrim( str_replace( "$this->source/", '', Utils\normalize_path( $this->main_file_path ) ), '/' )
```